### PR TITLE
SOC2 Doc Update

### PR DESCRIPTION
**Pull request by Travis-Wright-Systems**

### CODE REVIEW

**Comment by TravisGary** [comment]
This docs update goes with this blog post https://github.com/gravitational/web/pull/1998

File: docs/pages/enterprise/soc2.mdx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Teleport is designed to help satisfy two principal areas:
+CC6: Logical and Physical Access Controls 
+CC7 System Operations 
+
+Each principle has many “Points of Focus” which will apply differently to different products and organizations, talk to an auditor to understand exactly which points of focus apply.
+
+
+
+# What Key SOC2 Controls does Teleport help achieve?
+Teleport helps with 4 of the 9 control areas. We should write in the docs some specifics of which controls.
```
Comment (zmb3) [inline docs/pages/enterprise/soc2.mdx:24]: Would we want to remove the _We should write in the docs_ sentence before merging this?

File: docs/pages/enterprise/soc2.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+## CC6 Control Activities
+Teleport helps with separation of duties using RBAC and restricts access to authorized users
+- Provide role-based access controls (RBAC) using short-lived certificates and your existing identity management service.
+
+## CC6 Physical and Logical Access controls
+Teleport issues temporarily security credentials according to users role
```
Comment (zmb3) [inline docs/pages/enterprise/soc2.mdx:31]:
```suggestion
Teleport issues temporary security credentials according to the user's role.
```

**Comment by TravisGary** [comment]
Thanks @zmb3 I incorporated your suggestions

**Review by klizhentas — APPROVED** [review]

**Pull request merged**